src/Microsoft.AspNet.Authentication/DataHandler/TicketSerializer.cs

```diff
@@ -63,6 +63,17 @@ public static void Write([NotNull] BinaryWriter writer, [NotNull] Authentication
                         WriteWithDefault(writer, claim.Issuer, DefaultValues.LocalAuthority);
                         WriteWithDefault(writer, claim.OriginalIssuer, claim.Issuer);
                     }
+
+                    var bootstrap = identity.BootstrapContext?.ToString();
```

Comment:
Hum, if `BootstrapContext` is not a string, this may return the type name (which is rather useless). I think I'd probably ensure `BootstrapContext` is a string instead of using `ToString`.